.github/workflows/copy-completed-notebooks.yml

```
@@ -0,0 +1,117 @@
+name: Copy completed HTML notebooks to training website repository
+# Copy completed HTML notebooks from the training-modules repo to this repo
+# This action will open a pull request containing completed HTML notebooks
+
+# This is a manually triggered workflow that takes two inputs:
+#  # The name of the workshop being taught
+#  # The training-modules repo tag to copy notebooks from
+on:
+  workflow_dispatch:
+    inputs:
+      training-module:
+        type: choice
+        description: Use the dropdown menu to select which training workshop module this website is being created for.
+        options:
+          - "Introduction to R and Tidyverse"
+          - "Introduction to single-cell RNA-seq"
+          - "Advanced single-cell RNA-seq"
+        required: true
+      training-modules-tag:
+        type: string
+        description: The `training-modules` repo tag to copy completed notebooks from, e.g. 2023-june
+        default: "master"
+        required: true
+
+jobs:
+  file-completed-notebooks-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout training-modules repository
+        uses: actions/checkout@v3
+        with:
+          repository: AlexsLemonade/training-modules
+          ref: ${{ github.event.inputs.training-modules-tag }}
+          path: training-modules
+
+      - name: Checkout this repository
+        uses: actions/checkout@v3
+        with:
+          path: main
```

I think `main` is confusing here given that it usually refers to a branch. How about:
```suggestion
          path: website
```